README.md

```diff
@@ -136,7 +139,7 @@ To run the app alongside other related microservices see https://github.com/comm
 docker build -t communitiesuk/funding-service-design-post-award-data-store .
 docker run -p 8080:8080 communitiesuk/funding-service-design-post-award-data-store
 ```
-App should be available at `http://localhost:8080`
+App should be available at `http://localhost:4001`
```

Comment:
Looks like this was correct as is

If it's built and run as a standalone contain then it uses the port defined in the `Dockerfile` here
https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/f87dc108aa3ca9c6bdf51c0890bd5230994b1f2d/Dockerfile#L11

as opposed to the port defined in the docker-runner repo `docker-compose.yml` which is `4001`